pkgs/applications/graphics/photoflow/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, gettext, glib, libxml2, pkgconfig, swig, automake, gobjectIntrospection, cmake, libtiff, libjpeg, fftw, exiv2, lensfun, gtkmm2, libraw, lcms2, libexif, vips, expat, pcre, pugixml }:
+
+stdenv.mkDerivation {
+  name = "photoflow-0.2.8";
+
+  src = fetchFromGitHub {
+    owner = "aferrero2707";
+    repo = "PhotoFlow";
+    rev = "914561fcb1ca97de4814c4024d9a76e4b435c2af";
```

Comment:
If you extract a version into a separate variable, you can use `"v${version}"`.

Reply:
[Reportedly](https://github.com/aferrero2707/PhotoFlow/issues/175#issuecomment-371162490, the stable branch is more stable then the release. Could you use commit https://github.com/aferrero2707/PhotoFlow/commit/-f9bbea183fa02412d1d17075955d2284eeaf8174 and name the package `photoflow-unstable-03-06`?

Reply:
Did this. Release is quite old already so I hope this is better but for example gtk3 did more worse on this.

Reply:
Should there be also year in "version number"? I set package name to photoflow-unstable-2018-03-06 and amended commit message according.